webpack.common.js

```diff
@@ -121,7 +123,8 @@ module.exports = {
     new CopyPlugin([
       { from: 'theme-overrides/styles.css', to: 'theme-overrides/styles.css' },
       { from: 'src/demo-img', to: 'src/demo-img' },
-      { from: 'src/svg/flat/right-arrow.svg', to: 'src/demo-img' }
+      { from: 'src/svg/flat/right-arrow.svg', to: 'src/demo-img' },
+      // { from: 'src/svg/rich', to: 'svg/rich' } // Enable this line if you want to test svg as embedded image on dev environment and use following markup: <img class="msds-rich-icon" src="svg/rich/name-of-svg-file.svg" />
```

Comment:
You might want to keep it. It is intended to be a piece of informational code. It reminds you what needs to be done in case you want to test the svgs like in the main solution (as an embedded image), as I kept the original implementation in msds (still using svg as data URI)